OpenSim/Simulation/Wrap/WrapCylinder.cpp

```diff
@@ -32,7 +33,7 @@
 #include <OpenSim/Common/SimmMacros.h>
 #include <OpenSim/Common/Mtx.h>
 #include <sstream>
-
+#include "OpenSim/Simulation/Model/ModelDisplayHints.h"
```

Comment:
If adding headers are there some that can be removed? E.g. `SimmMacros.h`, `Mtx.h`, ...